### ISSUE TITLE
iot-nirvana: setup_gcp_environment.sh fails

### ISSUE THREAD

**Issue by prafullkotecha** [GoogleCloudPlatform/professional-services#85]
The script fails trying to create a boot disk, just before deleting the compute instance. The error description:
```
ERROR: (gcloud.compute.images.create) Could not fetch resource:
 - The disk resource 'projects/$MY_PROJECT/zones/us-central1-a/disks/debian9-java8-img' is already being used by 'projects/$MY_PROJECT/zones/us-central1-a/instances/debian9-java8-img'
```
My real project id replaced with $MY_PROJECT in this error message.

Possible Solution:
- Update the `startup_install_java8.sh` script to use `sudo` for installing `default-jre`
- `poweroff` command, that's in the script, didn't work when I tried it from the VM itself, by `ssh`ing into it.

**Comment by ryanmcdowell** [comment]
@prafullkotecha -- I think this was worked through offline with @dan-anghel. Can this be close or is there still an action item here?

**Comment by prafullkotecha** [comment]
@ryanmcdowell - Correct. @dan-anghel was super helpful in figuring out all the issues for my testing. But, the code still needs to be modified in order for it to work the first time.

**Comment by dan-anghel** [comment]
Added sudo in the JVM install and VM shutdown to fix this. Created pull request https://github.com/GoogleCloudPlatform/professional-services/pull/190 in

**Comment by ryanmcdowell** [comment]
Fixed in #190
